### PR TITLE
Wl admin invoice index

### PR DESCRIPTION
**Pull request by WilliamLampke**
user stories 32-35 with testing completed and ready to merge

### CODE REVIEW

File: spec/features/admin/invoice/show_spec.rb [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+    @transaction_1 = Transaction.create!(credit_card_number: '1', result: 0, invoice_id: @invoice_1.id)
+    @transaction_2 = Transaction.create!(credit_card_number: '1', result: 0, invoice_id: @invoice_1.id)
+  end
+  describe 'invoice show, has invoice info, invoice items info, and total revenue' do
+    it 'shows related information to a specific invoice' do
+      visit "/admin/invoices/#{@invoice_1.id}"
+      expect(page).to have_content("Id: #{@invoice_1.id}")
+      expect(page).to have_content('Status: completed')
+      expect(page).to have_content('Created at: Saturday, November 5, 2022')
```
Comment (DarbySmith) [inline spec/features/admin/invoice/show_spec.rb]: I think this test will fail when it is not November 5th. You will probably have to manually add the created_at date or use mock/stub here.

File: config/routes.rb [hunk 2]
```diff
@@ -1,8 +1,16 @@
 Rails.application.routes.draw do
+
+  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+    
   # admin
   get '/admin', to: 'admin#index'
+
   # merchants
-  get '/merchants/:merchant_id/dashboard', to: 'merchants#show'
+  # get '/merchants/:merchant_id/dashboard', to: 'merchants#show'
```
Comment (DarbySmith) [inline config/routes.rb]: Is this supposed to be commented out?

**Review by DarbySmith — COMMENTED** [review]
spec/features/admin/invoice/show_spec.rb: 25
You will probably have to refactor this test with a manually added created_at date for the invoice. 

config/routes.rb:9
I think the route needs to be uncommented

7 tests are failing (but they are due to the commented out route mentioned above)
coverage: 99.18% (when route uncommented)

File: spec/features/admin/invoice/show_spec.rb (hunk 1 above)
Comment (James-E-White) [reply spec/features/admin/invoice/show_spec.rb]: Updated with Darby and I on Zoom.

File: config/routes.rb (hunk 2 above)
Comment (James-E-White) [reply config/routes.rb]: It is commented out now.

**Comment by James-E-White** [comment]
Update all comments and conflicts with Darby and I on Zoom

**Review by James-E-White — APPROVED** [review]
Update all fixes with Darby and I on call

**Pull request merged**